package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
in my opinion `zat server -c .config.json` or `npm start` doesn't make much different